src/api/graphics.c

```diff
@@ -339,6 +339,15 @@ static int l_lovrGraphicsGetDimensions(lua_State* L) {
   return 2;
 }
 
+static int l_lovrGraphicsGetHasWindow(lua_State *L) {
```

Comment:
This should be called `l_lovrGraphicsHasWindow` (same with the platform function name) to stick with the existing naming conventions.